lib/kdump_utils.pm

```diff
@@ -397,6 +419,10 @@ sub full_kdump_check {
     if ($stage ne 'before') {
         ensure_serialdev_permissions;
     }
+    # We need to check bsc#1199326 after kdump_and_crash
+    if ($stage eq 'after') {
+        check_ssh_files();
```

Comment:
I have an idea about the solution: 
Maybe we can save this ssh file before migration and restore it after migration if needed. WDYT?

Reply:
that's a little bit complicated when this bug happens the .ssh dir sometimes was removed.
so a freshly created ssh key is the same as a backup and most important is it wil 100% work.

Reply:
In fact, I think the fresh created process is complicated. Anyway, this definitely work, NP.